app/billing/activities.go

```diff
@@ -21,6 +21,13 @@ type Activities struct {
 
 var a Activities
 
+const (
```

Comment:
Should maybe move to instrumentation package?

Reply:
Probably not needed, I think it's fine in here